File: syc98syc/Week3/01.16/PGS_완주하지 못한 선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        int index = participant.length-1;
        
        Arrays.sort(participant);
        Arrays.sort(completion);
        
        for(int i=0; i<participant.length-1; i++){
            if(!participant[i].equals(completion[i])){
                index=i;
                break;
            }
        }
        
        return participant[index];
    }
}

File: raeunlee/Week3/01.16/PGS_완주하지못한선수.py
def solution(participant, completion):
    answer = ''
    
    participant.sort()
    completion.sort()
    
    #순서대로 탐색했을 때, 발견하면 그 이름 return
    for i in range(len(completion)):
        if completion[i] != participant[i]:
            # 동명이인 이슈로 불일치 하는 이름을 찾으면 바로 break
            answer += participant[i]
            break
    
    if answer == '':
        answer += participant[-1]
        
    return answer